my_game.py

```diff
@@ -239,6 +241,57 @@ def update(self):
             self.kill()
 
 
+class StoppableEmitter():
+    """
+    It is possible to start and stop this emitter
+    """
+    def __init__(self,
+            target: arcade.Sprite,
+            particle_lifetime: float = 0.5,
+            noise: int = 15,
+            offset: Tuple[int] = (0, 6),
+            emit_interval: float = 0.01,
+            particle_count: int = 30,
+            start_alpa: int = 100):
```

Comment:
Ikke godt nok. Hvad i alverden er denne fejl. Hvem kan skrive alpa istedet for alpha. Altså helt seriøst.